project/AppDependencies.scala

```diff
@@ -0,0 +1,47 @@
+import play.core.PlayVersion
```

Comment:
Is moving the Dependencies in here the new standard?

Reply:
Yes @cavster , I think that it is, because that is how new projects get generated from https://github.com/hmrc/init-service/blob/master/templates/service/project/AppDependencies.scala , I'm in the process of documenting what I think the update process is, see my page "Upgrading library dependencies in a Play Framework 2.6 micro-service" a work in progress in my space of our Confluence